datastore/src/main/java/io/spine/server/storage/datastore/DsRecordStorage.java

```diff
@@ -168,14 +182,14 @@ public boolean delete(I id) {
     @Override
     protected Iterator<EntityRecord> readMultipleRecords(Iterable<I> ids,
                                                          FieldMask fieldMask) {
-        Function<Entity, @Nullable EntityRecord> transformer = input -> {
+        Function<@Nullable Entity, @Nullable EntityRecord> transformer = input -> {
             if (input == null) {
+                //noinspection ReturnOfNull
```

Comment:
Please use `@SuppressWarnings` instead.

Reply:
This is meant to fix IDEA bug. It should have been marked as warning because the output of the function _is_ annotated `@Nullable`.

Reply:
And I suppress the warning here and in the other place to avoid the ”yellow” mark in IDEA.